proposals/NNNN-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress.md

```diff
@@ -0,0 +1,140 @@
+# Adding new parameter of requiresAudioSupport and BluetoothDeviceAddress
+
+* Proposal: [SDL-NNNN](NNNN-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress.md)
+* Author: [Shohei Kawano](https://github.com/Shohei-Kawano)
+* Status: Awaiting review
+* Impacted Platforms: [ Core / iOS / Java Suite / RPC / HMI ]
+
+## Introduction
+
+In this proposal, by adding the `requiresAudioSupport` in `RegisterAppInterface` and the `BluetooothDeviceAddress` in `DeviceInfo`, when the device is connected via only USB, HU will connect to the SDL device's BT automatically or prompt the user requesting BT connection.  
+
+## Motivation
+
+Since Android recommends not supporting AOA2.0, Android will no longer be able to play audio using only USB, and it will be necessary to use BT A2DP to play audio.  
+In the current SDL Java Suite library, if requiresAudioSupport is TRUE and BT A2DP is not connected, SDL activation will be cancelled. Users aren't notified why the SDL App doesn't start, so the UX needs to be improved.
+To solve this problem, add `BluetoothDeviceAddress` and `requiresAudioSupport` so that a new SDL device can be specified in RegisterAppInterface, and prompt the user to connect the device via BT, or the HU will connect the device via BT automatically when connected via USB. 
```

Comment:
```suggestion
To solve this problem, add `BluetoothDeviceAddress` and `requiresAudioSupport` so that a new SDL device can be specified in `RegisterAppInterface`, and prompt the user to connect the device via BT, or the HU will connect the device via BT automatically when connected via USB. 
```